.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     docker:
       # this image is build from Dockerfile
       # https://github.com/pouchcontainer/pouchlinter/blob/master/Dockerfile
-      - image: pouchcontainer/pouchlinter:v0.2.1
+      - image: pouchcontainer/pouchlinter:v0.2.2
```

Comment:
you could update this line to be version of `v0.2.3`.
see https://github.com/pouchcontainer/pouchlinter/pull/19